docs/UserGuide.md

```diff
@@ -156,105 +156,160 @@ Examples:
+  
+  </div>
+  
+  Examples:
+  * `find */Burger` returns `CheeseBurger` and `fishburger`
+  * `find */cheese egg` returns `MacAndCheese` and `eggs`
 
-Format 2: `sort expiryDate`
 
-* `sort expiryDate` displays items in the inventory from the earliest expiry date to the latest.
-* Items without expiry date will be pushed to the back of the sorted list.
+### 3.7. Viewing expiring items : `reminder`  <a name = "reminder"></a>
 
-Examples:
+Want to know which items of yours are expiring soon? Then, this is the right command for you.<br>
 
-* `find Chocolate` followed by `sort quantity` sorts the items from the result of the `find` command.
-* `list Room 2` followed by `sort expiryDate` sorts the recorded list of items in Room 2 by expiry date.
+Format: `reminder NUMBER TIME_UNIT`
+<div markdown="block" class="alert alert-info">
 
-### Clearing all entries : `clear`
+**:information_source: Notes about the reminder command:**<br>
 
-Clears all entries from the location book.
+* `TIME_UNIT` must be either `days` or `weeks`. `day` and `week` **will not work**.
+*  Items without expiry date will not be shown.
+*  `NUMBER` **must be a positive integer** 1, 2 , 3, …​
 
-Format: `clear`
+</div>
 
-### Exiting the program : `exit`
+Examples:
+* `reminder 7 days` returns a list containing all items that are expiring within the next 7 days.
+* `reminder 2 weeks` returns a list containing all items that are expiring within the next 2 weeks.
+* Given today is 10 March 2021, and the inventory has 2 items: chocolate with expiry date of 13 March 2021 and 
+  banana with an expiry date of 20 March 2021.
+  * `reminder 5 days` returns the chocolate only.
+  * `reminder 2 weeks` returns both the chocolate and the banana.
 
-Exits the program.
+### 3.8. Sorting items : `sort` <a name = "sort"></a>
 
-Format: `exit`
+Want to see which items you are running out of or which items you have to consume soon? Use this command to find out!
 
-### Saving the data
+You can use this command in 2 different ways.
 
-StoreMando data are saved in the hard disk automatically after any command that changes the data. There is no need to
-save manually.
+* #### 3.8.1. Sorting items by quantity
+  Format 1: `sort quantity`
+  <div markdown="block" class="alert alert-info">
 
-### Editing the data file
+  **:information_source: Note about the sort by expiry date command:**<br>
 
-StoreMando data are saved as a JSON file `[JAR file location]/data/storemando.json`. Advanced users are welcome to
-update data directly by editing that data file.
+  * `quantity` is case-insensitive. Keying in `sort QUANTITY` is also a valid command.
+    <br></br>
 
-<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, StoreMando will discard all data and start with an empty data file at the next run.
-</div>
+  </div>
+  
+  Example:
+  * `sort quantity` sorts the items in the displayed list in ascending order of quantity.
```

Comment:
"sort quantity asc" is now the updated command for sorting items in ascending order of quantity. There is also the "sort quantity desc" command for sorting items in descending order of quantity. You may want to update this! :)